pages/kubernetes/how-to/use-sfs-with-kubernetes.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: How to use SFS with Kubernetes Kapsule
+description: This page explains how to use the Scaleway File Storage Container Storage Interface (CSI) driver to enable Kubernetes users to manage Scaleway File Storage volumes within their clusters.
+tags: kubernetes kubernetes-kapsule kapsule sfs
+dates:
+  validation: 2025-07-18
+  posted: 2025-07-18
+categories:
+  - containers
+  - kubernetes
+---
+import Requirements from '@macros/iam/requirements.mdx'
+
+The Scaleway File Storage Container Storage Interface (CSI) driver enables Kubernetes users to manage Scaleway File Storage volumes within their clusters.  
+The Scaleway File Storage CSI driver is designed to work with Kubernetes Kapsule and Kosmos clusters, providing a standardized interface to create, manage, and attach file storage volumes to your containerized workloads. For more details on Scaleway File Storage, refer to the [Scaleway File Storage documentation](https://www.scaleway.com/en/file-storage/).
+
+## Supported features
+
+The Scaleway File Storage CSI driver supports the following features:
+
+- Dynamic provisioning: Automatically create Scaleway File Storage volumes using PVCs and StorageClasses.
+- `ReadWriteMany` access mode: Allows multiple pods to read from and write to the same file storage volume simultaneously.
+- Import of existing Volumes: Integrate pre-existing Scaleway File Storage volumes into Kubernetes.
+- Custom storage classes: Define specific parameters, such as file system type (e.g., ext4), for volumes.
+
+<Requirements />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Created](/kubernetes/how-to/create-cluster/) a Kubernetes Kapsule cluster
+- Helm installed for deploying the CSI driver
+- Access to the Scaleway File Storage API
+
+## Installation
```

Comment:
It's not needed to install the CSI driver when using our managed Kubernetes.

During the File Storage product beta, customers can use a magic tag to pre-install it automatically. I'm not sure if we want to publicly communicate this tag yet.